mypy/fastparse.py

```diff
@@ -582,12 +582,14 @@ def visit_Set(self, n: ast35.Set) -> Node:
     # ListComp(expr elt, comprehension* generators)
     @with_line
     def visit_ListComp(self, n: ast35.ListComp) -> Node:
-        return ListComprehension(self.visit_GeneratorExp(cast(ast35.GeneratorExp, n)))
+        g = self.visit_GeneratorExp(cast(ast35.GeneratorExp, n))
+        return ListComprehension(cast(GeneratorExpr, g))
```

Comment:
`visit_GeneratorExp` returns a `GeneratorExpr`.  I think this is related to changing the type of the `with_line` decorator.